src/main/scala/com/typesafe/sbt/SbtGit.scala

```diff
@@ -117,7 +117,19 @@ object SbtGit {
     gitDescribedVersion := gitReader.value.withGit(_.describedVersion).map(v => git.gitTagToVersionNumber.value(v).getOrElse(v)),
     gitCurrentTags := gitReader.value.withGit(_.currentTags),
     gitCurrentBranch := Option(gitReader.value.withGit(_.branch)).getOrElse(""),
-    gitUncommittedChanges in ThisBuild := gitReader.value.withGit(_.hasUncommittedChanges)
+    gitUncommittedChanges in ThisBuild := gitReader.value.withGit(_.hasUncommittedChanges),
+    scmInfo := {
+      val remote = """origin[ \t]+(?:git@|https?\:\/\/)([^:\/]+)[:\/](.*)\.git[ \t]+\(fetch\)""".r
+      
+      Process("git remote -v").lines_!.collect {
```

Comment:
Fair enough. I made a commit to change this and also refactored the code a bit, because the regex started to look quite ugly.